front/src/reducers/osrdconf/types.ts

```diff
@@ -161,6 +162,11 @@ export type PacedTrainId = string & { readonly __type: unique symbol };
 
 export type TrainId = TrainScheduleId | OccurrenceId;
 export type TimetableItemId = TrainScheduleId | PacedTrainId;
+export type TimetableItemToEditData = {
+  timetableItemId: TimetableItemId;
+  originalPacedTrain?: PacedTrainWithDetails;
+  occurrenceId?: OccurrenceId;
+};
```

Comment:
Hum I've thinking, it might be better to have this type, what do you think ?
```suggestion
export type TimetableItemToEditData =
  | {
      timetableItemId: TrainScheduleId;
    }
  | {
      timetableItemId: PacedTrainId;
      originalPacedTrain: PacedTrainWithDetails;
      occurrenceId?: OccurrenceId;
    };
```

Reply:
Done. Had to update some stuff (in the `upsert exceptions in paced train to update` commit) in `formatPacedTrainPayload` to narrow down the proper kind of `timetableItemToEditData`